packages/shared/src/chunker.ts

```diff
@@ -0,0 +1,49 @@
+// No clue why, but 3600 matches 4kb in the browser
```

Comment:
These tabs are huge no? :D

Reply:
Just in general though I'm not super clear how chunking works? Could you document the approach in the file itself?

Also I would prefer if this went into gotrue-js, I can see it being generally useful.

Reply:
Yeah it would make sense if this was in `gotrue-js`. Would it be ok to get this released in the auth-helpers first and then work our way back to `gotrue-js` after?